src/mixins/java/org/spongepowered/common/mixin/core/world/level/dimension/DimensionTypeMixin.java

```diff
@@ -45,4 +58,22 @@ public static Path getStorageFolder(ResourceKey<Level> worldKey, Path defaultLev
         // Sponge Start - The directory is already set to be at this location
         return defaultLevelDirectory;
     }
+
+    @Override
+    public DimensionType bridge$decorateData(final SpongeWorldTypeTemplate.SpongeDataSection data) {
```

Comment:
As I said above, I don't think this is needed anymore as long as Vanilla stores this in their CODEC. Let me know if they don't.

Reply:
does not seem like it. the constructor used defaults it to false